SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -172,6 +172,9 @@ const Schema_FieldId UNREAL_RPC_PAYLOAD_OFFSET_ID = 1;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_INDEX_ID = 2;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_PAYLOAD_ID = 3;
 const Schema_FieldId UNREAL_RPC_PAYLOAD_TRACE_ID = 4;
+const Schema_FieldId UNREAL_RPC_EVENT_TRACE_UNIQUE_ID = 5;
+
+const Schema_FieldId UNREAL_COMPONENT_EVENT_DATA_FIELD = 18999; // 1 less than FComponentIdGenerator::RESERVED_COMPONENT_ID_START;
```

Comment:
Interested in the choice of ID

Reply:
It won't overlap with generated field ids from the schema compiler!

Reply:
It could also be RESERVED_COMPONENT_ID_END + 1